### PR TITLE
fix ignores for @qooxdoo/compiler

### PR DESCRIPTION
**Pull request by hkollmann**
tthis will remove `qx.dev.unit.Sinon: [4335,23] Unresolved use of symbol setTimeout.clock`

### CODE REVIEW

**Review by cboulanger — DISMISSED** [review]

**Comment by cboulanger** [comment]
@johnspackman Would there be a way that the compiler could ignore these kind of symbols by default (global vars, NodeJS API), so that this doesn't have to be done manually each time?

**Comment by hkollmann** [comment]
Please not. In Webapps node vars are forbidden. I just integrated a global ignores section in compile.json for this purpose.

**Review by oetiker — DISMISSED** [review]

**Review by level420 — DISMISSED** [review]

**Review by johnspackman — DISMISSED** [review]

**Review by cboulanger — DISMISSED** [review]

File: framework/source/class/qx/dev/unit/Sinon.js [hunk 1]
```diff
@@ -74,8 +70,10 @@ qx.Bootstrap.define("qx.dev.unit.Sinon",
 
 
 /**
- * @ignore(module, require, global, process.*, setImmediate)
+ * @ignore(module, require, global, process.*, setImmediate.*)
```
Comment (derrell) [inline framework/source/class/qx/dev/unit/Sinon.js]: `setImmediate` is a function, not an object with members. Why does it need `.*` here? Seems wrong.

File: framework/source/class/qx/dev/unit/Sinon.js [hunk 2]
```diff
@@ -74,8 +70,10 @@ qx.Bootstrap.define("qx.dev.unit.Sinon",
...
  * @ignore(msSetImmediate.*)
+ * @ignore (process.*) 
+ * @ignore (setTimeout.*)
```
Comment (derrell) [inline framework/source/class/qx/dev/unit/Sinon.js]: Ditto.

**Review by derrell — CHANGES_REQUESTED** [review]

**Review by cboulanger — APPROVED** [review]

File: framework/source/class/qx/dev/unit/Sinon.js (hunk 1 above)
Comment (hkollmann) [reply framework/source/class/qx/dev/unit/Sinon.js]: Done

**Review by derrell — COMMENTED** [review]
I just looked carefully at this file to see what's going on. Firstly, msSetImmediate is not used anywhere in the file, so that `@ignore` can be removed entirely. 

Secondly, it's referencing `setTimeout.clock`. I've never heard of `setTimeout` having a `clock` property, and neither has the interweb. What the heck is this code doing?!?

**Comment by hkollmann** [comment]
I really do not know. It's from the qooxdoo used sinon package. I only want to get away the warnings during compile.

**Review by derrell — APPROVED** [review]
This file needs review to figure out what it's doing. In the mean time, the proposed change doesn't hurt, and will remove the warning.

**Review by johnspackman — APPROVED** [review]

**Review by oetiker — APPROVED** [review]

**Review by level420 — APPROVED** [review]

**Comment by level420** [comment]
full quorum. merging
thank you @hkollmann

**Pull request merged**